plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterActive.php

```diff
@@ -159,7 +158,7 @@ private function render_chip_item( $args ) {
 
 		$remove_label = sprintf( 'Remove %s filter', wp_strip_all_tags( $title ) );
 		?>
-		<li class="wc-block-active-filters__list-item">
+		<li class="list-item">
 			<span class="is-removable wc-block-components-chip wc-block-components-chip--radius-large">
```

Comment:
The only thing I left alone were these components chip classes. I assume they're borrowing styles from another component with some conditional overrides.